src/shogun/preprocessor/KernelPCA.cpp

```diff
@@ -213,5 +192,3 @@ CDenseFeatures<float64_t>* CKernelPCA::apply_to_string_features(CFeatures* featu
 
```

Comment:
Is this an overloaded method? It doesnt make sense indeed....in particular since the code can be shared no? (The kernel matrix is all we need). Maybe this is actually a copy-paste error? :D